run-tests.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+generate_id() {
+  LC_ALL=C tr -dc 'a-z' </dev/urandom | head -c 13 ; echo
+}
+
+port=9000
+containers=()
+sonarqube_container_name="it-sonarqube"
+network="it-network"
+
+print_usage() {
+  cat <<EOF
+usage: $0 [IMAGE...]
+
+examples:
+       $0 "sonar-scanner-cli:4.1"
+EOF
+}
+
+info() {
+  echo "[info] $@"
```

Comment:
shellcheck.net recommends to not mix arrays with strings, so write like this:
```
info() {
  echo "[info] $*"
}
```

Same comment for the other display functions. (Not to be confused with `"$@"` and `"${arr[@]}"` which is different, and not to be replaced blindly by `"$*"` and `"${arr[*]}"`. Let me know if you need clarification.)

Reply:
ok